app/controllers/foreman_bootdisk/disks_controller.rb

```diff
@@ -40,6 +40,23 @@ def full_host
       end
     end
 
+    def subnet
+      begin
+        subnet = @disk.try(:subnet)
```

Comment:
Holy cup of :coffee: 

Any comments for the ones below? Going to send the new PR today.